From: ayaben <abenjerrad@gmail.com>
Date: Sat, 24 Oct 2026 04:33:00 +0000
Subject: Translated using Weblate (Arabic)

Currently translated at 100.0% (12 of 12 strings)

Translation: openkylin-huanghe/vpn
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/vpn/ar/
---
 plugins/vpn/translations/ar.ts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/plugins/vpn/translations/ar.ts b/plugins/vpn/translations/ar.ts
index 7743cad..e851cd8 100644
--- a/plugins/vpn/translations/ar.ts
+++ b/plugins/vpn/translations/ar.ts
@@ -28,12 +28,12 @@
         <location filename="../vpn.ui" line="53"/>
         <location filename="../vpn.cpp" line="70"/>
         <source>VPN</source>
-        <translation></translation>
+        <translation>VPN</translation>
     </message>
     <message>
         <location filename="../vpn.ui" line="68"/>
         <source>import</source>
-        <translation></translation>
+        <translation>import</translation>
     </message>
     <message>
         <location filename="../vpn.cpp" line="159"/>
